### DISCUSSION TITLE
Docs should use correct term for the hash symbol

### DISCUSSION THREAD

**Discussion by jwortmann** [typst/typst#1777] [General]
In the documentation pages, there are several places where the [*hash*](https://en.wikipedia.org/wiki/Number_sign), or *number sign* (`#`) is confusingly called a "hashtag". Per definition, a [*hashtag*](https://en.wikipedia.org/wiki/Hashtag) is composed of a hash symbol, followed by a word or letter sequence.

The docs pages should use the correct term *hash* instead of *hashtag*. I didn't provide a PR because the term "hashtag" is also accidentally used a few times within the source code, and I'm unsure whether it would be justified to use the correct term there as well: https://github.com/search?q=repo%3Atypst%2Ftypst+hashtag&type=code

**Comment by lorenzleutgeb** [comment]
Hey! I just came across this discussion. I was on the brink of duplicating it. I second @jwortmann. I also considered filing a PR, but I refrained from doing so for the same reason.

I stumbled while reading the following paragraph in the [typst docs (Tutorial &middot; Writing in Typst)](https://typst.app/docs/tutorial/writing-in-typst/):

> Within the argument list of the `figure` function, Typst is already in code mode. This means, you can now remove the hashtag before the image function call. The hashtag is only needed directly in markup (to disambiguate text from function calls).

For me, `#typst` is a "hashtag" (see [Wikipedia](https://en.wikipedia.org/wiki/Hashtag)), while `#` is a "hash" (also "hash symbol" or "number symbol", see [Wikipedia](https://en.wikipedia.org/wiki/Number_sign)). So, a hashtag is composed of the hash symbol, followed by some alphanumeric characters.

And while you may consider this pedantic, I do think it is important to make this distinction. [Searching for "hashtag" in typst/typst quickly surfaces multiple occasions of this confusion](
https://github.com/search?q=repo%3Atypst%2Ftypst+hashtag&type=code).

I'd be interested to hear from the authors whether this is deliberate. Maybe we can discuss changing this. Thank you!

**Reply by laurmaedje** [reply]
I don't really mind either way to be honest. I'm open for a PR.

**Comment by laurmaedje** [comment]
Fixed.

**Reply by lorenzleutgeb** [reply]
In f474639ebe6f1bc7d5ab398ae1df16af48c77708, that is. Thanks :)
